### PR TITLE
Fix:(README.md)some grammatical misktakes and simplified it 

### PR DESCRIPTION
**Pull request by Adarshjaiss**
I have done some minor grammatical changes in readme file, and simplified some sentences which was creating a little bit problem from the contributors to understand.

## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#commit-messages.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/NativeScript/blob/master/tools/notes/DevelopmentWorkflow.md#running-unit-tests-application.
- [ ] Tests for the changes are included - https://github.com/NativeScript/NativeScript/blob/master/tools/notes/WritingUnitTests.md.

## What is the new behavior?

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

**Comment by Adarshjaiss** [comment]
@cla-bot check

File: README.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ We love you and PR's 🤗 Please follow our [contributing guide](https://github.
 - [NativeScript on Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript)
 - [NativeScript documentation](https://docs.nativescript.org/)
 
-## Other framework source repositories
+## our Other frameworks sources repositories:-
```
Comment (NathanWalker) [inline README.md:55]: Thank you @Adarshjaiss for these improvements. Everything looks great, although this one sounds a bit funny, how about just:
```
## More source repos
```

If you don't mind making that one adjustment we'd love to get this merged.

Comment (Adarshjaiss) [reply README.md:55]: @Nathan Walker Thank you sir, for mentoring and reviewing this code. Actually, I have done that silly mistake, I am sorry for that. I am just correcting it. 🙂

Comment (NathanWalker) [reply README.md:55]: You’ll notice in the changeset above in green, no further change has been pushed yet. Perhaps you made the change locally but haven’t committed and pushed it yet?

**Review by NathanWalker — CHANGES_REQUESTED** [review]

**Comment by Adarshjaiss** [comment]
i am sorry for that i have troubled you sir. but now i have fixed all changes and the pr is ready to go.

**Review by NathanWalker — APPROVED** [review]

**Pull request merged**